modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/shared/components/last-updated-info/MetricsCalculatedInfo.es.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayLayout from '@clayui/layout';
+import React from 'react';
+
+import moment from '../../util/moment.es';
+
+export default function MetricsCalculatedInfo({infoDate}) {
+	const date = infoDate
+		? moment.utc(infoDate).format(Liferay.Language.get('mmm-dd-hh-mm-a'))
+		: null;
+
+	return (
+		date && (
+			<div className="updated-info-container">
+				<ClayLayout.ContainerFluid className="mt-3">
```

Comment:
Hey @ramonmsp,

we cannot create the message concatenating the strings in that way`{`${Liferay.Language.get('metrics-calculated')} ${date}`}` since there are languages with a inverse written order(RTL) where this approach will fail.

You can follow this example of usage:

https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/workflow-builder/WorkflowStep.es.js#L116-L121

https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/app-builder-workflow/app-builder-workflow-lang/src/main/resources/content/Language.properties#L55

Thx